src/Transition/InputRulesFiles/Rules8-9-0-to-9-0-0.md

```diff
@@ -35,3 +35,25 @@ Move up all later numeric fields by one (old N4-N10 become new N3-N9).
 
 
 
+Input Changes
+=============
+
+This file documents the structural changes on the input of EnergyPlus that could affect interfaces, etc.
+This was previously an Excel workbook that made for very difficult version control, especially during busy times around code freezes.
+
+# Object Change: `AirflowNetwork:Distribution:Component:OutdoorAirFlow`
```

Comment:
Need to code the transition for this before merging. I'll do that.

Reply:
@mjwitte Thanks.